Makefile

```diff
@@ -190,12 +192,15 @@ test-cluster:
 OPENSHIFT_VERSIONS?="v4.7"
 CHANNELS="stable,stable-${LOGGING_VERSION}"
 DEFAULT_CHANNEL="stable"
-generate-bundle: regenerate $(OPM)
-	MANIFEST_VERSION=${LOGGING_VERSION} OPENSHIFT_VERSIONS=${OPENSHIFT_VERSIONS} CHANNELS=${CHANNELS} DEFAULT_CHANNEL=${DEFAULT_CHANNEL} hack/generate-bundle.sh
-.PHONY: generate-bundle
-
-bundle: generate-bundle
-.PHONY: bundle
+bundle: $(OPM) $(shell find manifests -type f)
+	@rm -rf bundle
+	$(MAKE) regenerate
+	MANIFEST_VERSION=${LOGGING_VERSION} \
+	OPENSHIFT_VERSIONS=${OPENSHIFT_VERSIONS} \
+	CHANNELS=${CHANNELS} \
+	DEFAULT_CHANNEL=${DEFAULT_CHANNEL} \
+	hack/generate-bundle.sh
+	@touch $@
```

Comment:
@jcantrill @vparfonov: **Note**: Since CLO is using a modern version of the `operator-sdk` you should replace `opm` with the `operator-sdk` bundle commands.

Reply:
@vparfonov what's  holding up your merge? I'd much rather commit the preview version work **after** your changes.
Is it still  just minor lint issues? If your sure they're harmless then maybe get your changes in immediately and we sort out lint later...